tests/orders.test.js

```diff
@@ -0,0 +1,50 @@
+// tests/orders.test.js
+const { create, get, list, edit } = require('../orders');
+const orderData = require('../data/order1.json');
+const { create: createProduct } = require('../products');
+const productData = require('../data/product1.json');
+
+describe('Orders Module', () => {
+let createdProduct;
+let createdOrder;
+
+beforeAll(async () => {
+createdProduct = await createProduct(productData);
+orderData.products = [createdProduct._id];
+});
```

Comment:
This should be using the[ `productTestHelper` ](https://github.com/Abhishekgovindhu/is-5600-week-08/blob/main/tests/test-utils/productTestHelper.js)as described in the lab instructions